### PR TITLE
change on FCN.py

### PR DESCRIPTION
**Pull request by yeLer**
In view of the inability to converge in the loss of training, I made the following changes to the FCN.py file:
1 The initial learning rate is 1e-4, and the training iteration is 1e5 times, which becomes the original learning rate of 5e4 per iteration.
2 The loss function changes from sparse_softmax_cross_entropy_with_logits() to fcn's custom loss function.

Experiments show that the modified code can finally converge and have a good effect on the verification set.

### CODE REVIEW

**Comment by WANGZEQI123** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
